Gopkg.toml

```diff
@@ -1,10 +1,10 @@
 [[constraint]]
   name = "github.com/gogo/protobuf"
-  version = "0.5.0"
+  version = "1.0.0"
 
 [[constraint]]
-  branch = "master"
   name = "github.com/golang/protobuf"
+  version = "1.0.0"
```

Comment:
This has to change. We can go as high as 1.3.0, or we could stop using protobuf entirely.